titles/rosa_cli/master.adoc

```diff
@@ -0,0 +1,9 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+// CHANGE TITLE HERE
+= rosa CLI
```

Comment:
Some of our CLIs, such as OpenShift, are referred to as "OpenShift CLI (`oc`)". I don't know if your comment to change title here means this is still going to need to be updated? Have you considered, "ROSA CLI (rosa)"?

Reply:
Well, ROSA is the acronym for the product name, which we're not supposed to use in title. "rosa" is the name of the actual CLI. So, being between a rock and a hard place here, I'm opting to go with the actual name of the CLI, otherwise I need to name this chunk (book) "Red Hat OpenShift Service on AWS CLI" and I think CLI gets lost in that. Also, I believe there are a lot of places that refer to `oc` CLI, so I feel good about my decision ;-)